src/components/Tree/utils/mocks.tsx

```diff
@@ -141,6 +141,7 @@ export const mockNodesFlat: DraggableItem<TreeFlatListItem>[] = [
         icon: <IconFolder size={IconSize.Size16} />,
         value: "https://weare.frontify.com/document/923#/test",
         sort: null,
+        badge: <IconSmileysPeople size={IconSize.Size16} />,
```

Comment:
Looove the smily :D 
We also have the Badge Component, maybe we could include also the Badge in another Node to also show this showcase.